projects/components/src/toggle-switch/toggle-switch.component.test.ts

```diff
@@ -0,0 +1,40 @@
+import { fakeAsync } from '@angular/core/testing';
+import { FormsModule } from '@angular/forms';
+import { MatSlideToggle, MatSlideToggleChange, MatSlideToggleModule } from '@angular/material/slide-toggle';
+import { createHostFactory, Spectator } from '@ngneat/spectator/jest';
+import { ToggleSwitchComponent } from './toggle-switch.component';
+
+describe('Toggle Switch Component', () => {
+  let spectator: Spectator<ToggleSwitchComponent>;
+
+  const createHost = createHostFactory({
+    component: ToggleSwitchComponent,
+    shallow: true,
+    imports: [MatSlideToggleModule, FormsModule]
```

Comment:
If I mock MatSlideToggle I won't be able to test the inputs as this component is using get/set. Hence I am importing the entire module.